server/anp/search_providers/talpa_video.py

```diff
@@ -42,6 +41,7 @@ class TalpaVideoSearchProvider(superdesk.SearchProvider):
 
     def __init__(self, provider):
         super().__init__(provider)
+        self.url = provider.get('config', {}).get('url', '')
```

Comment:
just wondering how many urls are available here, maybe an env variable would be enough for it

Reply:
Yes, that could be an option, but the use case was to make it possible to change the URL from the UI as mentioned in the ticket.